stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/DOM.java

```diff
@@ -450,6 +460,17 @@ public int getColor() {
     public String objectGroup;
   }
 
+  private static class SetAttributesAsTextRequest {
+    @JsonProperty(required = true)
+    public int nodeId;
+
+    @JsonProperty(required = true)
+    public String text;
+
+    @JsonProperty
+    public String name;
```

Comment:
do we use this? if it's not required as per the JSON protocol documentation, then I bet we can just omit it